front/lib/api/assistant/feedback.ts

```diff
@@ -26,8 +29,82 @@ export type AgentMessageFeedbackType = {
   userId: number;
   thumbDirection: AgentMessageFeedbackDirection;
   content: string | null;
+  isConversationShared: boolean;
+  agentConfigurationVersion: number;
+  agentConfigurationId: string;
+  createdAt: Date;
 };
 
+export type AgentMessageFeedbackWithMetadataType = AgentMessageFeedbackType & {
+  conversationId: string;
+  userName: string;
+  userImageUrl?: string;
+};
+
+export async function getAgentConfigurationFeedbacks({
+  auth,
+  agentConfigurationId,
```

Comment:
[SECURITY] As per my comment below you want to take AgentConfigurationType here or you want to call getAgentConfiguration() before pulling the feedback